mule-extensions-api/src/main/java/org/mule/runtime/extension/api/manifest/DescriberManifest.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.extension.api.manifest;
+
+import org.mule.runtime.extension.api.introspection.declaration.spi.Describer;
+import org.mule.runtime.extension.api.manifest.ExtensionManifestBuilder.DescriberManifestBuilder;
+
+import java.util.Map;
+
+/**
+ * Manifest that enunciates the main properties of a {@link Describer}
+ * to be instantiated by the Mule Runtime.
+ * <p>
```

Comment:
intellij started to generate `<p>` instead of `</p>` when we moved to java 8. Have learned not to contradict intellij. It probably has its reasons